exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -1,193 +1,196 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 
 namespace TriviaGame
 {
     public class Game
     {
-        List<string> players = new List<string>();
+        readonly List<string> players = new List<string>();
+        static int maxPlayerCount = 6;
 
-        int[] places = new int[6];
-        int[] purses = new int[6];
+        readonly  int[] playerPosition = new int[maxPlayerCount];
+        readonly  int[] playerScores = new int[maxPlayerCount];
 
-        bool[] inPenaltyBox = new bool[6];
+        readonly  bool[] inPenaltyBox = new bool[maxPlayerCount];
+        readonly  int maximumBoardGameSpaces = 12;
 
-        LinkedList<string> popQuestions = new LinkedList<string>();
-        LinkedList<string> scienceQuestions = new LinkedList<string>();
-        LinkedList<string> sportsQuestions = new LinkedList<string>();
-        LinkedList<string> rockQuestions = new LinkedList<string>();
+        readonly LinkedList<string> popQuestions = new LinkedList<string>();
+        readonly LinkedList<string> scienceQuestions = new LinkedList<string>();
+        readonly LinkedList<string> sportsQuestions = new LinkedList<string>();
+        readonly LinkedList<string> rockQuestions = new LinkedList<string>();
 
-        int currentPlayer = 0;
+        int currentPlayer;
         bool isGettingOutOfPenaltyBox;
+        readonly int questionsCountPerCategory = 50;
+        readonly int winningScore = 6;
 
         public Game()
         {
-            for (int i = 0; i < 50; i++)
+            for (int questionIndex = 0; questionIndex < questionsCountPerCategory; questionIndex++)
             {
-                popQuestions.AddLast("Pop Question " + i);
-                scienceQuestions.AddLast(("Science Question " + i));
-                sportsQuestions.AddLast(("Sports Question " + i));
-                rockQuestions.AddLast(CreateRockQuestion(i));
+                popQuestions.AddLast(CreateQuestion("Pop", questionIndex));
+                scienceQuestions.AddLast(CreateQuestion("Science", questionIndex));
+                sportsQuestions.AddLast(CreateQuestion("Sports", questionIndex));
+                rockQuestions.AddLast(CreateQuestion("Rock", questionIndex));
             }
         }
 
-        public string CreateRockQuestion(int index)
+        public string CreateQuestion(string categoryName, int index)
         {
-            return "Rock Question " + index;
-        }
```

Comment:
I like that you've brought this out into a generic "CreateQuestion" method.

(Adding new comment because I couldn't edit my previous one correctly)